docs-website/docusaurus.config.js

```diff
@@ -54,7 +54,7 @@ module.exports = {
           position: "right",
         },
         {
-          to: "docs/demo",
```

Comment:
can we also delete the demo.md page?

Reply:
hmm okay, i was about to leave it for future use cases but tbh I think it would be find without it.